service/history/replication/stream_sender.go

```diff
@@ -256,21 +263,24 @@ func (s *StreamSenderImpl) sendCatchUp() (int64, error) {
 		s.clientShardKey.ShardID,
 	)
 
+	catchupEndExclusiveWatermark := s.shardContext.GetQueueExclusiveHighReadWatermark(tasks.CategoryReplication).TaskID
```

Comment:
when will QueueExclusiveHighReadWatermark be updated/persisted for replication queue?